popup.js

```diff
@@ -1,6 +1,18 @@
 var inputHandle = document.getElementById("githubHandle");
 var btn = document.getElementById("check");
 
+var handle = document.getElementById("githubHandle").value;
+if (handle === "") {
+	chrome.storage.sync.get("lastSearched", function (data){
+		var storedHandle = data['lastSearched'];
+  	  if (storedHandle != "" && storedHandle != null) {
+	    console.log(storedHandle);
+		document.getElementById("githubHandle").value = storedHandle;
+		getData(storedHandle);
```

Comment:
What about storing this data also instead of getting it every time ? Although this is a much cleaner way but will saving the data be better ?

Reply:
See below - but, it's your project, if you would like it to store the data I could create another PR for that?

Reply:
I think it is better this way for now.  We can do it later on as I was thinking we should save thumbnails for all searched users. We can think about it at that time.